test/visreg/visreg.ts

```diff
@@ -11,7 +11,7 @@ export function suite(browserName: string) {
     test.before(async () => {
         const config: IConfig = {
             browser: browserName,
-            screenshotsDir: "visreg/screenshots",
+            screenshotsDir: "artifacts/visregScreenshots",
```

Comment:
I wonder....should this be `artifacts/screenshots/visreg`?

I'm fine as-is, I think it looks good....I could do either.

Reply:
I'm indifferent.